package.json

```diff
@@ -34,6 +34,7 @@
     "fast-deep-equal": "^3.1.3",
     "mixin-deep": "^2.0.1",
     "mz": "^2.7.0",
+    "object-assign-deep": "^0.4.0",
```

Comment:
Could you remove the added dependencies here? After that this can be merged.

Reply:
Forgot to git add ...
Done